server/src/main/java/org/opensearch/search/aggregations/bucket/DocCountProvider.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Looks like this is the last file missing the OpenSearch license provisions

Reply:
ah... sorry